config.py

```diff
@@ -128,6 +130,9 @@
             "Pfsense": "PFSense",
             "Vyos": "VyOS"
         },
+        "proxmox": {
+            "ProxmoxVE": "PVE",
```

Comment:
It should be `"Pve": "ProxmoxVE",` and need to rerun `autogen.sh` to update alias for proxmox. This might be the last request change.